Makefile

```diff
@@ -289,8 +295,8 @@ uninstall: manifests kustomize ## Uninstall CRDs from the K8s cluster specified
 
 .PHONY: deploy
 deploy: manifests kustomize docker-push-kind ## Deploy controller to the K8s cluster specified in ~/.kube/config.
```

Comment:
adding `$(CLUSTERCTL)` or `clusterctl` as dependency to the `deploy` target would be convenient.

Reply:
Thanks for the suggestion. Done!